security/security_profiles_operator/spo-release-notes.adoc

```diff
@@ -12,6 +12,50 @@ These release notes track the development of the Security Profiles Operator in {
 
 For an overview of the Security Profiles Operator, see xref:../security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-7-1"]
+== Security Profiles Operator 0.7.1
+
+The following advisory is available for the Security Profiles Operator 0.7.1:
+
+* link:https://access.redhat.com/errata/RHSA-2023:2029[RHSA-2023:2029 - OpenShift Security Profiles Operator bug fix update]
+
+[id="spo-0-7-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Security Profiles Operator (SPO) now automatically selects the appropriate `selinuxd` image for RHEL 8- and 9-based RHCOS systems.
++
+[IMPORTANT]
+====
+Users that mirror images for disconnected environments must mirror both `selinuxd` images provided by the Security Profiles Operator.
+====
+
+* You can now enable memory optimization inside of an `spod` daemon. For more information, see xref:../../security/security_profiles_operator/spo-advanced.adoc#spo-memory-optimization_spo-advanced[Enabling memory optimization in the spod daemon].
++
+[NOTE]
+====
+SPO memory optimization is not enabled by default.
+====
+
+* The daemon resource requirements are now configurable. For more information, see xref:../../security/security_profiles_operator/spo-advanced.adoc#spo-daemon-requirements_spo-advanced[Customizing daemon resource requirements].
+
+* The priority class name is now configurable in the `spod` configuration. For more information, see xref:../../security/security_profiles_operator/spo-advanced.adoc#spo-custom-priority-class_spo-advanced[Setting a custom priority class name for the spod daemon pod].
+
+[id="spo-0-7-1-deprecations"]
+=== Deprecated and removed features
+
+* The default `nginx-1.19.1` seccomp profile is now removed from the Security Profiles Operator deployment.
+
+[id="spo-0-7-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, a Security Profiles Operator (SPO) SELinux policy did not inherit low-level policy definitions from the container template if you selected another template, such as `net_container`. The policy would not work because it required low-level policy definitions that only existed in the container template. This issue occurred when the SPO SELinux policy attempted to translate SELinux policies from the SPO custom format to the Common Intermediate Language (CIL) format. With this update, the container template appends to any SELinux policies that require translation from SPO to CIL. Additionally, the SPO SELinux policy can inherit low-level policy definitions from any supported policy template. (link:https://issues.redhat.com/browse/OCPBUGS-12879[*OCPBUGS-12879*])
```

Comment:
A full stop is needed for below sentence:
`Previously, a Security Profiles Operator (SPO) SELinux policy did not inherit low-level policy definitions from the container template.`
And a comma needed after the `such as `net_container``, 
`If you selected another template, such as `net_container`, the policy would not work because it required low-level policy definitions that only existed in the container template.`

Reply:
Thank you, that does read better :)